src/main/java/org/jfree/chart/date/SerialDate.java

```diff
@@ -346,7 +352,7 @@ public static int monthCodeToQuarter(int code) {
      *
      * @return a string representing the supplied month.
      */
-    public static String monthCodeToString(int month) {
+    public static String monthCodeToString(@IntRange(from=1, to=12) int month) {
```

Comment:
Inconsistent spacing; compare to line 324 above.  Either always, or never, put spaces around `=`.  Probably always, but I don't care which you choose so long as it is consistent.